src/expressServer.js

```diff
@@ -140,6 +140,11 @@ const socketServer = new Server(server);
 const setupSocket = require('./live');
 const io = socketServer.of('/'); // Create an instance of Socket.io
 
+// Add this route to serve the order notifications page
+app.get('/order-notifications', (req, res) => {
+  res.render('orderNotifications'); // Render the EJS template
+});
+
```

Comment:
Express reads routes and middleware from top-to-bottom.

So when you type in `localhost:8000/order-notifications` in the browser, it's gonna check in this file each line that says `app.use` or `app.<httpMethod>`. 

*   it doesn't match anything from line 119-127
*   it doesn't find a matching file in the `public` static folder
*   next thing it gets to is `app.use(notFoundMiddleware)` on line 131. That middleware runs for anything that hasn't been matched up to that point, which is why you get the "Route does not exist" page. That's why we say that that one and the errorHandlerMiddleare "must be defined after all other routes and middleware".